packages/transport/src/modules/accounts/IdentityDeletionProcessController.ts

```diff
@@ -27,6 +27,23 @@ export class IdentityDeletionProcessController extends TransportController {
         return this;
     }
 
+    public async loadNewIdentityDeletionProcessFromBackbone(identityDeletionProcessId: string): Promise<IdentityDeletionProcess> {
+        const identityDeletionProcessJSONResponse = await this.identityDeletionProcessClient.getIdentityDeletionProcess(identityDeletionProcessId);
+
+        const newIdentityDeletionProcess = this.createIdentityDeletionProcessFromBackboneResponse(identityDeletionProcessJSONResponse);
+        await this.identityDeletionProcessCollection.create(newIdentityDeletionProcess);
+        this.eventBus.publish(new IdentityDeletionProcessStatusChangedEvent(this.parent.identity.address.toString(), newIdentityDeletionProcess));
+        return newIdentityDeletionProcess;
+    }
+
+    public async updateIdentityDeletionProcessById(identityDeletionProcess: string): Promise<IdentityDeletionProcess> {
```

Comment:
IMO this can just be called updateCache like in the other controllers that handle caching